client/my-sites/plugins/main.jsx

```diff
@@ -41,6 +39,14 @@ import PluginNotices from 'lib/plugins/notices'
 import JetpackManageErrorPage from 'my-sites/jetpack-manage-error-page'
 import PlanNudge from 'components/plans/plan-nudge'
 import FeatureExample from 'components/feature-example'
+import SectionHeader from 'components/section-header'
+import ButtonGroup from 'components/button-group'
+import Button from 'components/button'
+import Gridicon from 'components/gridicon'
+import SelectDropdown from 'components/select-dropdown'
+import DropdownItem from 'components/select-dropdown/item'
+import DropdownSeparator from 'components/select-dropdown/separator'
```

Comment:
hmmmm... I may be wrong, but I've seen it done like this in other files and I just assumed that it was already a convention... maybe not, because it's not written anywhere